### PR TITLE
feat(rootfs/deploy.py): remove image after it's pushed

### PR DESCRIPTION
**Pull request by arschles**
Fixes https://github.com/deis/dockerbuilder/issues/29

### CODE REVIEW

File: rootfs/deploy.py [hunk 1]
```diff
@@ -67,3 +67,6 @@ def log_output(stream, decode):
 print("pushing to registry")
 stream = client.push(registry+'/'+imageName, tag=imageTag, stream=True, insecure_registry=True)
 log_output(stream, False)
+print("deleting image")
+stream = client.remote_image(image=registry+'/'+imageName, force=True, noproune=False)
```
Comment (bacongobbler) [inline rootfs/deploy.py]: `remote_image`? Should this be `remove_image`?

Also `noprune` rather than `noproune`

Comment (carmstrong) [inline rootfs/deploy.py]: Maybe it's French

Comment (bacongobbler) [inline rootfs/deploy.py]: Always happy to see more international contributions to the project :fr:

Comment (arschles) [inline rootfs/deploy.py]: haha, nope - it's supposed to be `remove_image`. fixed in 28db3ddcdba773cedad6f57f327adb7ac37f8bd7

File: rootfs/deploy.py [hunk 2]
```diff
@@ -67,3 +67,6 @@ def log_output(stream, decode):
 print("pushing to registry")
 stream = client.push(registry+'/'+imageName, tag=imageTag, stream=True, insecure_registry=True)
 log_output(stream, False)
+print("deleting image")
+stream = client.remove_image(image=registry+'/'+imageName, force=True, noproune=False)
```
Comment (bacongobbler) [inline rootfs/deploy.py]: still need to change `noproune` to `noprune` :)

Comment (arschles) [inline rootfs/deploy.py]: I fail at spelling. Fixed in 5c2558f

**Comment by arschles** [comment]
Waiting to merge until e2e tests go green

**Comment by arschles** [comment]
@smothiki should this stay open? needs rebase and tests fail

**Comment by smothiki** [comment]
Let's push this in for this release. If the image pull is successful, we dont need a copy of the image on the host

**Pull request merged**